brambling/templates/brambling/forms/customformfield.html

```diff
@@ -4,18 +4,46 @@
 {% formconfig row using "brambling/forms/rows/bootstrap.html" %}
 
 {% block rows %}
-	{% formrow form.name %}
-	{% formrow form.field_type %}
-	{% formrow form.default %}
-	{% formrow form.required %}
-	{% formrow form.help_text %}
-	{% formrow form.choices %}
-	<div class='row js-hide'>
-		<div class='col-md-6'>{% formrow form.index %}</div>
-		<div class='col-md-6'>{% formrow form.DELETE %}</div>
+	<div class='row'>
+		<div class='col-sm-1 hidden-xs'>
+			<span class="js-sortable-handle btn btn-link btn-block a-secret pull-right"><i class="fa fa-fw fa-bars"></i></span>
+		</div>
+		<div class='margin-trailer-half-xs col-sm-4'>
+			{% formrow form.name using "brambling/forms/rows/no_label.html" with placeholder="Do you snore?" %}
```

Comment:
I don't love having this as the placeholder for every field but I might let it go till we have a superior form editing interface.